src/logic/safe/store/actions/createTransaction.ts

```diff
@@ -107,20 +107,13 @@ export class TxSender {
 
   // On transaction completion (either confirming or executing)
   async onComplete(signature?: string, confirmCallback?: ConfirmEventHandler): Promise<void> {
-    const { txArgs, safeTxHash, txProps, dispatch, notifications, isFinalization, approveAndExecute = false } = this
+    const { txArgs, safeTxHash, txProps, dispatch, notifications, isFinalization } = this
```

Comment:
You can remove `approvedAndExecute` passed to `processTransaction` and all occurances of it throughout the `process`/`createTransaction functions.